pkg/promtail/api/entry_parser.go

```diff
@@ -14,13 +15,20 @@ type EntryParser int
 
 // Different supported EntryParsers.
 const (
+	CRI    EntryParser = iota
```

Comment:
can you instead add CRI as the last element in the list, and there should only be one `EntryParser = iota`

Reply:
@slim-bean Done